app/gui/view/welcome-screen/src/side_menu.rs

```diff
@@ -0,0 +1,73 @@
+//! Side menu for Welcome Screen.
+//!
+//! Side menu contains a list of available projects and "new project" button.
+
+use ensogl::prelude::*;
+
+use crate::ClickClosure;
+use crate::Frp;
+
+use ensogl::system::web;
+use ensogl::system::web::NodeInserter;
+use web_sys::Element;
+
+
+
+// ================
+// === SideMenu ===
+// ================
+
+/// Side menu for Welcome Screen. Contains a list of available projects and "new project" button.
+#[derive(Clone, CloneRef, Debug)]
+pub struct SideMenu {
+    logger:             Logger,
+    /// Root DOM element of the side menu.
+    pub root:           Element,
+    new_project_button: Element,
+    projects_list:      Element,
+    closures:           Rc<RefCell<Vec<ClickClosure>>>,
+}
+
+impl SideMenu {
+    /// Constructor.
+    // TODO [vitvakatu]: _frp argument is not used now, but will be in the next tasks.
```

Comment:
Remove the comment and remove the unused argument. We do not know what will happen in the next tasks and there is probability that it will not be fixed. If it will be needed, you'll add it I the next PRs.

Reply:
Unused argument removed